dotCMS/src/main/java/com/dotcms/rest/api/v1/index/ESIndexResource.java

```diff
@@ -374,6 +374,9 @@ public Response snapshotIndex(@Context HttpServletRequest request,
     }
 
 
+    /**
+     * @deprecated  As of 2016-10-12, replaced by {@link #snapshotIndex(request, inputFile, inputFileDetail ,params)} and {@link #snapshotIndex(request, inputFile, inputFileDetail ,params)}
+     */
```

Comment:
You might want to annotate the method as a deprecated, in the future we will have a way to return a header to the user that calls a deprecated api, in order to mention witch is the new one.